medcat/utils/model_utils.py

```diff
@@ -0,0 +1,372 @@
+import math
+import torch
+from torch import nn
+import sys
+
+BertLayerNorm = torch.nn.LayerNorm
```

Comment:
Is this module even used? relation_extraction.models (rightly so) uses these classes imported from transformers.models etc. Any code that isn't used can you remove and resubmit for review

Reply:
this has now been removed.